RUNBOOK.md

```diff
@@ -12,7 +12,7 @@ Production
 
 ## Primary URL
 
-https://www.ft.com/__origami/service/navigation-data/v2/navigation.json
+https://origami-navigation-data.in.ft.com/v2/navigation.json
```

Comment:
What does Primary URL mean exactly? The Origami consumer-facing URL is `https://www.ft.com/__origami/service/navigation-data/v2/navigation.json` and `https://origami-navigation-data.in.ft.com/v2/navigation.json` is really some kind of implementation detail of ours.

Reply:
The primary URL is the URL that consumers of the service would use to access it. In the case of navigation data there's only one consumer: the navigation service. The navigation service accesses the data via `https://origami-navigation-data.in.ft.com/v2/navigation.json` making it the primary URL.

As an aside, the reason I spotted and changed this is because Ops pointed out that this URL: https://www.ft.com/__origami/service/navigation-data/v2/navigation.json no longer works

Reply:
(changed here https://github.com/Financial-Times/origami-navigation-service/pull/75/files)

Reply:
Ah this is the navigation-data, I misread it as navigation-service 🙈